src/components/AgmExtension/ExtensionRequest.vue

```diff
@@ -209,14 +203,36 @@ export default class ExtensionRequest extends Vue {
     ]
   }
 
+  /** Whether the extension request is valid or not */
+  get extensionRequestValid (): boolean {
+    const isFirstAgmValid = this.data.isFirstAgm && !!this.data.agmYear
+    const isSubsequentAgmValid = (
+      this.data.isFirstAgm === false &&
```

Comment:
I specifically check `=== false` a few times in this file. That's because some values in the `data` object can be `null`, but that doesn't mean they are valid. 

For example, the answers to "Is this the first AGM?" can be "Yes" or "No", with values `true` and `false` respectively. To check `isSubsequentAgmValid`, then the "No" field must be ticked (which evaluates to false). However when neither field is ticked (not a valid extension request), it remains `null`. Therefore there can be no ambiguity between null and false.

<img width="608" alt="Screenshot 2023-11-17 at 1 22 49 PM" src="https://github.com/bcgov/business-filings-ui/assets/122323255/3398ed30-5bc2-43c9-b4ca-5921595176b8">